cylc/flow/network/schema.py

```diff
@@ -2103,6 +2103,8 @@ class Meta:
              - ``started`` implies ``submitted``.
              - ``succeeded`` and ``failed`` imply ``started``.
              - custom outputs and ``expired`` do not imply any other outputs.
+
+            Valid for: paused, running, stopping workflows.
```

Comment:
I don't think this command is really valid for stopping workflows:

```suggestion
            Valid for: paused, running.
```

Reply:
If waiting for tasks to finish while stopping, is it not valid for users to set task states? I think it can be done from the CLI

Reply:
Ooh, there is a reasonable point there due to this use case:

>  I need to orphan a "stuck" job submission
>
> -- https://github.com/cylc/cylc-admin/blob/master/docs/proposal-interventions.md#4-i-need-to-orphan-a-stuck-job-submission

Which could otherwise block shutdown.

Ok, leave as is.